packages/wrangler/src/pages/build-env.ts

```diff
@@ -40,9 +41,10 @@ export const Handler = async (args: PagesBuildEnvArgs) => {
 			EXIT_CODE_NO_CONFIG_FOUND
 		);
 	}
+	logger.log("Reading build configuration from your wrangler.toml file...");
```

Comment:
will this show up in the CI logs?

Reply:
These will, if we print the stdout of this commend. They won't show up by default

Reply:
yes we do

Reply:
I think `build configuration` might be confusing here. perhaps `environment configuration` might be better

Reply:
It includes the build output directory as well, which is why I called it build configuration